duckduckgo/iOS
# Site Rating / Privacy Grade Report

Reviewer: Chris
Asana:  https://app.asana.com/0/414709148257752/425269760045532

**Description**:
Generates a site rating / privacy grade report for the top 500 sites. There is also a Jenkins job (currently pointing at this branch) to allow us to run the report https://jenkins.duckduckgo.com/job/ios%20-%20duckduckgo%20app%20-%20generate%20site%20report/

**Steps to test this PR**:
1. Edit the `top500_sites.json` so it only has a few entries
2. Change the XCode scheme to `TopSitesReport` and run the associated tests
3. Watch the logs for the output file name
4. Open the file and ensure the data has been saved

###### Reviewer Checklist:


###### PR DRI Checklist:
